lib/grandpa/grandpa.go

```diff
@@ -507,6 +507,9 @@ func (s *Service) playGrandpaRound() error {
 
 	logger.Debugf("sending pre-vote message %s...", pv)
 	roundComplete := make(chan struct{})
+	// roundComplete is a signal channel which is closed when the round completes
+	// (will receive the default value of channel's type), so we don't need to
+	// explicitly send a value.
```

Comment:
nit I think that would be enough of a comment

```suggestion
	// roundComplete is a signal channel which is closed when the round completes
```